test/test-execute/exec-temporaryfilesystem-usr.service

```diff
@@ -12,4 +12,4 @@ ExecStart=/bin/sh -c '! touch /usr/hoge'
 ExecStart=/bin/sh -c '! touch /usr/bin/hoge'
 
```

Comment:
This test could also verify that some files in /usr/bin can be accessed.

Reply:
Thanks. More tests are added.